nfc/Kconfig

```diff
@@ -4,58 +4,10 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-config NRFXLIB_NFC
-	bool
-	prompt "Enable NFC"
-	select NRFX_NFCT
-	help
-	  Enable NFC functionality
-
-if NRFXLIB_NFC
-
-comment "Common NFC module configuration"
-
-config NFC_T2T_LIB_ENABLED
+config NFC_T2T_NRFXLIB
 	bool
 	prompt "Enable NFC Type 2 Tag library"
```

Comment:
On the occasion, you could also get rid of these `prompt`s here.
```
bool "Enable NFC Type 2 Tag library"
```

Reply:
Ok changed.